src/main/java/org/mitre/synthea/helpers/TimeSeriesData.java

```diff
@@ -0,0 +1,48 @@
+package org.mitre.synthea.helpers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TimeSeriesData {
+  /** Provides a collection of time series values as well as sampling metadata
+   *  concerning those values.
+   */
+  private List<Double> values;
+  private double period; // number of seconds between samples
```

Comment:
> number of seconds between samples

If this is really seconds, and not say, milliseconds, then the FHIR exporters might be wrong.

In `State.java`, you have the following code:

```java
      // Series data default period is 1.0s
      if (period <= 0.0) {
        period = 1.0;
      }
```

The `series_data.json` example that I uploaded produces the following FHIR R4 `Observation.valueSampledData`

```json
        "valueSampledData": {
          "origin": {
            "value": 0.0,
            "system": "http://unitsofmeasure.org"
          },
          "period": 1.0,
          "factor": 1.0,
          "lowerLimit": 0.0,
          "upperLimit": 10.0,
          "dimensions": 1,
          "data": "1 2 3 4 4 3 2 1 1 2 3 4 "
        }
```

But according to http://hl7.org/fhir/R4/datatypes-definitions.html#SampledData.period, `SampledData.period` is measured in **milliseconds**

Reply:
Doh! Yes you are correct. I don't know where I got seconds from. It also confirms milliseconds is the correct measurement in https://www.hl7.org/fhir/datatypes.html#SampledData, which is the document I used as a reference.

Will fix that now.

Reply:
Oh wait I remember this now. My thought was that the TimeSeriesData object should store in seconds since I personally think that's generally easier to work with than milliseconds. The SBSCL simulations use seconds as their form of measurement for example.\

As far as whether the user should input seconds or milliseconds, that can be modified in the State definition. I personally think it is more straightforward to enter seconds rather than milliseconds, but if you think most module developers would be more comfortable with milliseconds, we can change it.

But yes that also means we need to do the appropriate conversion when exporting with FHIR, which I messed up. I'll change that to properly convert to milliseconds.